tests/test_config_generation.py

```diff
@@ -24,7 +24,7 @@ def _test_configs(folder_name, **kwargs):
     kubetools_config = load_kubetools_config(app_dir, **kwargs)
 
     with mock.patch('kubetools.kubernetes.config.job.uuid4', lambda: 'UUID'):
-        services, deployments, jobs = generate_kubernetes_configs_for_project(
+        services, deployments, jobs, cronjobs = generate_kubernetes_configs_for_project(
```

Comment:
Maybe due to the gitignore but I was expecting to see an update to one of the test kubetool configs to add cron?

Reply:
I added the following test file `tests/configs/basic_app/k8s_cronjobs.yml`